### PR TITLE
Milestone one: Feature animals table

### PR DESCRIPTION
**Pull request by darikmohammed**
# Features 
## Create a table named animals with the following columns:

- id: integer
- name: string
- date_of_birth: date
- escape_attempts: integer
- neutered: boolean
- weight_kg: decimal

-SQL is copied in schema.sql file. 

## Following data are inserted in the animal table

- Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
- Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
- Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

- SQL is copied in data.sql .

## Queries for the following quests:

- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
- SQL is copied in queries.sql.

### Screenshot of the result 

Screen Shot 2022-08-09 at 1 11 58 AM

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    ID INT GENERATED ALWAYS AS IDENTITY, 
+    NAME TEXT NOT NULL,
```
Comment (Ismailco) [inline schema.sql]: - [x] Kindly consider using varchar for name, e.g ` name varchar(255),`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE NAME LIKE '%mon';
+SELECT * FROM animals WHERE DATE_OF_BIRTH BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Ismailco) [inline queries.sql]: - [x] list the name of all animals born between 2016 and 2019, kindly select only names not`*`, also the second date should be `2019/31/12`

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT NAME FROM animals WHERE NEUTERED AND ESCAPE_ATTEMPTS< 3;
+SELECT DATE_OF_BIRTH FROM animals WHERE NAME IN ('Agumon' , 'Pikachu');
+SELECT NAME, ESCAPE_ATTEMPTS FROM animals WHERE WEIGHT_KG > 10.5;
+SELECT * FROM animals WHERE NEUTERED;
```
Comment (Ismailco) [inline queries.sql]: - [x] You can specify the value of neutered in the query like so: ``SELECT * FROM animals WHERE neutered = 'true';``

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * FROM animals WHERE NAME = 'Gabumon';
```
Comment (Ismailco) [inline queries.sql]: - [x] Here you should find all animals that name is not Gabumon e.g ``SELECT * FROM animals WHERE name <> 'Gabumon';``

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @darikmohammed, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @darikmohammed, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**